CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # CHANGELOG
 
+## 2.0.0 (2022-11-14)
+
+MLflow 2.0.0 includes several breaking changes from the 1.x API, additional major features and improvements
+
+Breaking Changes:
+
+- [Recipes] Renamed `mlflow.pipelines` to `mlflow.recipes`
+- [Tracking] Remove deprecated `list` APIs for experiments, models, and runs from Python, Java, JS, and R APIs (#6785, #6786, #6787, #6788, #6800, #6868, @dbczumar)
```

Comment:
```suggestion
- [Tracking] Remove deprecated `list` APIs for experiments, models, and runs from Python, Java, and R APIs (#6785, #6786, #6787, #6788, #6800, #6868, @dbczumar)
```

Nit: JS isn't relevant here, since users don't interact with it directly